src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useMemo } from 'react';
+import classNames from 'classnames';
+import { Errors } from '../types/Errors';
+
+interface Props {
+  error: Errors | '';
+  setError: (value: Errors | '') => void;
+}
+
+export const ErrorNotification: React.FC<Props> = ({ error, setError }) => {
+  const errorMessage = useMemo(() => {
+    switch (error) {
+      case Errors.LoadError:
+        return Errors.LoadError;
+      case Errors.EmptyTitle:
+        return Errors.EmptyTitle;
+      case Errors.AddTodoError:
+        return Errors.AddTodoError;
+      case Errors.DeleteTodoError:
+        return Errors.DeleteTodoError;
+      case Errors.UpdateTodoError:
+        return Errors.UpdateTodoError;
+      default:
+        return '';
+    }
```

Comment:
I'm sure you can find a better way for the errors handling than such a switch case.